rep-0003.rst

```diff
@@ -169,14 +200,12 @@ These targets, including starting and ending support dates, are based on the Dis
 C++
 ---
 
-We use the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
+As of ROS Jade, we are still using the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
 While we mainly develop with gcc, no use of compiler-specific features is allowed
 without proper use of macros to allow use on other platforms.
 
-Use of C++[0|1]x or tr1 features are only allowed if support for that feature is checked
-at compile time, and equivalent functionality exists without requiring C++[0|1]x
-code.  A wholesale jump to C++[0|1]x will not happen until all commonly used
-OS platforms fully support it.
+Use of C++11/C++14 features and filesystem/networking/etc... TS's (Technical Specifications) is allowed if they are checked for at configure time and equivalent functionality can be provided with the extra compiler features.
```

Comment:
@wjwwood:  I might be mistaken, but shouldn't this be
" [...] if they are checked for at configure time and equivalent functionality can be provided **without** the extra compiler features."?

Reply:
Yes.

Reply:
https://github.com/ros-infrastructure/rep/pull/152